bundle/src/main/java/com/cognifide/cq/cqsm/foundation/permissions/PermissionActionHelper.java

```diff
@@ -22,114 +22,129 @@
 import com.cognifide.cq.cqsm.core.utils.MessagingUtils;
 import com.cognifide.cq.cqsm.foundation.permissions.exceptions.PermissionException;
 import com.cognifide.cq.cqsm.foundation.permissions.utils.JackrabbitAccessControlListUtil;
-
-import org.apache.commons.lang.StringUtils;
-import org.apache.jackrabbit.api.security.JackrabbitAccessControlList;
-
 import java.security.Principal;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-
+import javax.jcr.PropertyType;
 import javax.jcr.RepositoryException;
 import javax.jcr.Value;
 import javax.jcr.ValueFactory;
+import javax.jcr.ValueFormatException;
 import javax.jcr.security.AccessControlManager;
 import javax.jcr.security.Privilege;
+import org.apache.commons.lang.StringUtils;
+import org.apache.jackrabbit.api.security.JackrabbitAccessControlList;
 
 public class PermissionActionHelper {
 
-	private static final String STRICT = "STRICT";

```

Comment:
we should define code styling for APM and format whole source code with it to make it consistent